### ISSUE TITLE
Change language

### ISSUE THREAD

**Issue by tlygnersjo** [jimmiw/php-time-ago#83]
How would i set Swedish language?

Danish translation seems to work:
```
  $filter = new Twig_SimpleFilter('timeago', function ($datetime) {
    $timeago = new Westsworld\TimeAgo( new \Westsworld\TimeAgo\Translations\Da() );

    return $timeago->inWords( new DateTime( $datetime ) );
  });
```

But not Swedish (or any other translation class with underscore in its class name):
```
  $filter = new Twig_SimpleFilter('timeago', function ($datetime) {
    $timeago = new Westsworld\TimeAgo( new \Westsworld\TimeAgo\Translations\Sv_SE() );

    return $timeago->inWords( new DateTime( $datetime ) );
  });
```

What am i doing wrong? @jimmiw

**Comment by tlygnersjo** [comment]
This is the error that i got:
`Fatal error: Uncaught Error: Class 'Westsworld\TimeAgo\Translations\Sv_SE' not found`

**Comment by tlygnersjo** [comment]
I solved this by renaming **src/WestsWorld/TimeAgo/Translations/Sv_SE.php** to **Se.php** and in that file change the class name from **Sv_SE** to **Se**

When calling this localized and renamed class i do the following:
```
  $filter = new Twig_SimpleFilter('timeago', function ($datetime) {
    $timeago = new Westsworld\TimeAgo( new \Westsworld\TimeAgo\Translations\Se() );

    return $timeago->inWords( new DateTime( $datetime ) );
  });
```


Is this a bug? @jimmiw

**Comment by jimmiw** [comment]
Hello, this is now solved in 3.1.0 package.
I switched from psr-0 to psr-4, so try to checkout that version and remember to run composer install. I also added a test for swedish language, so we can make sure this will not happen again :)

Thanks for reporting the issue
